README.md

```diff
@@ -7,9 +7,13 @@ This is a Java wrapper of the [libidpasslite](https://github.com/idpass/idpass-l
 ![id front](testdata/idpass-lite-java-sample-front.png?raw=true "front") ![id back](testdata/idpass-lite-java-sample-back.png?raw=true "back")
 
 
-## Building
+## Building for Java `idpass-lite-java-${VERSION}.jar`
 ```bash
-./gradlew build
+./build.sh jar
```

Comment:
The script just call ./gradlew build, so let's keep it as it ease. no need to add abstraction